packages/core/README.md

```diff
@@ -1,6 +1,18 @@
-# Theia - Core extension
+<div align='center'>
 
-See [here](https://www.theia-ide.org/doc/index.html) for a detailed documentation.
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>THEIA - CORE EXTENSION</h2>
+
+<hr />
+
+</div>
+
+## Description
+
+The `@theia/core` extension is the main extension which provides functionality, base widgets, and contribution points for all dependent extensions. At a high-level, the extension allows for all dependent extensions the ability to contribute additional functionality to the application. For example, an application can contribute commands, menu items, statusbar items, new dialogs, and new widgets all with the help of `@theia/core`.
```

Comment:
most importantly it provides application and shell APIs as well as contributions to hook into application lifecycles.

Reply:
> most importantly it provides application and shell APIs as well as contributions to hook into application lifecycles.

Thank you, I wanted some help to describe `@theia/core` (there was no formal description before) in a way that would help extension developers.